app/assets/javascripts/discourse/app/lib/uppy/upload-debugging.js

```diff
@@ -54,80 +59,80 @@ export default Mixin.create({
       }
     }
     return measureResult;
-  },
+  }
 
-  _instrumentUploadTimings() {
-    if (!this._performanceApiSupport()) {
+  instrumentUploadTimings(uppy) {
+    if (!this.#performanceApiSupport()) {
       warn(
-        "Some browsers do not return a PerformanceMeasure when calling this._performanceMark, disabling instrumentation. See https://developer.mozilla.org/en-US/docs/Web/API/this._performanceMeasure#return_value and https://bugzilla.mozilla.org/show_bug.cgi?id=1724645",
+        "Some browsers do not return a PerformanceMeasure when calling this.#performanceMark, disabling instrumentation. See https://developer.mozilla.org/en-US/docs/Web/API/this.#performanceMeasure#return_value and https://bugzilla.mozilla.org/show_bug.cgi?id=1724645",
```

Comment:
hmmmm is this a find/replace mistake?

https://developer.mozilla.org/en-US/docs/Web/API/this.#performanceMeasure#return_value

doesn't exist

Reply:
`this._performanceMark` was a function in this UploadDebugging class, and it's now renamed to `this.#performanceMark`

Reply:
I know but you also changed the link in the doc:

```
See https://developer.mozilla.org/en-US/docs/Web/API/this.#performanceMeasure#return_value
```

And this link doesn't exist:
<img width="1181" alt="Screenshot 2024-10-17 at 10 50 43" src="https://github.com/user-attachments/assets/ca3a4156-3991-473c-96c9-5ce55a65c6ce">

Reply:
I see the problem now, the previous link was also wrong. It should be:

https://developer.mozilla.org/en-US/docs/Web/API/PerformanceMeasure#return_value

Reply:
Ahhhh I see, thanks! Will fix